settings.py

```diff
@@ -52,7 +52,7 @@
         'ATOMIC_REQUESTS': True,
         'TEST': {
             'CHARSET': 'utf8',
-            'COLLATION': 'utf8_general_ci',
+            'COLLATION': 'utf8_distinct_ci',
```

Comment:
The test runner fails to apply the collation migration with a MySQL `OperationalError` so I'm trying some other approaches.